packages/London-Boroughs 2 (Alternative)/README.md

```diff
@@ -0,0 +1,50 @@
+# London, Boroughs
```

Comment:
What is the difference with other packages? 🤔 
I would prefer to publish only 1 package with the right data.

Moreover all the previous comments apply to this package too.